src/gql/utils/consts.test.ts

```diff
@@ -3,43 +3,25 @@ import {
   queryBatchHandler,
   GqlOutCommunityPool,
   communityPoolQueryString,
-  GqlOutDelegations,
-  delegationsQueryString,
   arg,
 } from ".."
 
 describe("queryBatchHandler tests", () => {
   test("queryBatchHandler", async () => {
     const resp = await queryBatchHandler<{
       communityPool: GqlOutCommunityPool[]
-      delegations: GqlOutDelegations[]
     }>(
-      [
-        communityPoolQueryString({}, true),
-        delegationsQueryString(
-          {
-            limit: 1,
-          },
-          true
-        ),
-      ],
+      [communityPoolQueryString({}, true)],
```

Comment:
This test was here to validate the combination of multiple queries, can we add staking back here or a different query?

Reply:
https://github.com/NibiruChain/ts-sdk/blob/4cda626ed72e824be727d680a42947d6ebdcdff0/src/gql/heart-monitor/heart-monitor.test.ts#L732

Looks like we already have a test of queries combination

Reply:
That particular test is part of the heartmonitor and tests that the heartmonitor function call works (which inevitably calls this function)

This directly tests the function for coverage

Reply:
done